llvm/utils/TableGen/Common/CodeGenTarget.cpp

```diff
@@ -211,10 +211,9 @@ void CodeGenTarget::ReadInstructions() const {
 
   // Parse the instructions defined in the .td file.
   for (const Record *R : Insts) {
-    auto &Inst = Instructions[R];
+    auto &Inst = InstructionMap[R];
     Inst = std::make_unique<CodeGenInstruction>(R);
```

Comment:
nit: since we're changing this part of the code, should we just use a normal `insert` or `try_emplace` instead?